.travis.yml

```diff
@@ -1,20 +0,0 @@
-language: ruby
```

Comment:
imo lint configs are misleading if not connected to CI. can we hook that up (here or in a later commit)?

Reply:
Yes, we can/should add a GitHub Actions to lint MD, JS, CSS, etc.

Reply:
BTW, I created an issue to add the deploy workflow as well #113